queries.sql

```diff
@@ -27,3 +27,86 @@ SELECT * FROM ANIMALS WHERE NAME != 'Gabumon';
 */
 SELECT * FROM ANIMALS WHERE WEIGHT_KG BETWEEN 10.4 AND 17.3;
 
+
+/* 
+
+  transactions 
+
+*/
+
+/* Update the animals table by setting the species column to unspecified. Verify that change was made. Then rolled back the change and verified that the species columns went back to the state before the transaction. */
+
+BEGIN;
+UPDATE ANIMALS SET SPECIES = 'unspecified';
+SELECT * FROM ANIMALS;
+ROLLBACK;
+
+/* Inside a transaction:
+Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+Verify that changes were made.
+Commit the transaction. */
+
+BEGIN;
+UPDATE ANIMALS SET SPECIES = 'digimon' WHERE NAME LIKE '%mon';
+UPDATE ANIMALS SET SPECIES = 'pokemon' WHERE SPECIES IS NULL OR SPECIES = '';
+SELECT * FROM ANIMALS;
+COMMIT;
```

Comment:
* Also here You are [required](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20changes%20persist%20after%20commit.) to Verify that changes persist after commit, Kindly check the review comments above for Hints ⬆️